gcb/build.yaml

```diff
@@ -0,0 +1,59 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+timeout: 7200s
+steps:
+- name: 'gcr.io/cloud-builders/git'
+  dir: 'go/src/k8s.io'
```

Comment:
Shouldn't this be `/workspace/go/src/k8s.io` ?

Reply:
@listx -- either would work, I think.

`dir` is set relative to `/workspace`.
ref: https://cloud.google.com/cloud-build/docs/build-config